src/contracts/factory/FundRanking.sol

```diff
@@ -0,0 +1,29 @@
+pragma solidity ^0.4.21;
+pragma experimental ABIEncoderV2;
+
+import "./FundFactory.sol";
+import "../fund/accounting/Accounting.sol";
+
+contract FundRanking {
+    function getFundDetails(address _factory)
```

Comment:
@gempi this is the function you'll need to call in melon.js in order to get the ranking